exercises/js/inputHint.js

```diff
@@ -0,0 +1,50 @@
+class SearchHandler {
+  constructor(options) {
+    this.parentForm = $(options.formSelector);
+    this.className = options.className;
+    this.inputField = this.parentForm.find(options.inputSelector);
+    this.labelField = this.parentForm.find(options.labelSelector);
+  }
+
+  init() {
+    this.removeLabelField();
+    this.bindEventHandlers();
+  }
+
+  removeLabelField() {
+    // Set the value of the search input to the text of the label element
+    // Add a class of "hint" to the search input
+    this.setLabelText();
+    // Remove the label element
+    this.labelField.remove();
+  }
+
+  setLabelText() {
+    const labelText = this.labelField.text();
+    this.inputField.val(labelText);
```

Comment:
As I said earlier, create a method,
setInputFieldValue(value) { this.inputField.val(value) }
and use it here